src/initializers/koa/add-request-context.ts

```diff
@@ -1,7 +1,8 @@
 import { AsyncLocalStorage } from 'async_hooks';
 import { Context, Middleware } from 'koa';
+import { pick } from 'lodash';
 
-export default function(als: AsyncLocalStorage<Map<string, any>>): Middleware {
+export default function(als: AsyncLocalStorage<Map<string, any>>, config): Middleware {
```

Comment:
Could you also convert this to a named function to resolve [this](https://github.com/Workable/orka/issues/261)?